rules/default.js

```diff
@@ -50,10 +49,7 @@ module.exports = function (options) {
 
     heading : function (text, level, raw) {
         var result = {
-            elem : 'header',
-            mods : { // BUG: MUST BE elemMods
-                level : level
-            },
+            elem : 'h' + level,
```

Comment:
I like to put  all techs into elems folders, for example I put all headers css in to content__header